docker/server/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8
+LABEL maintainer="Democracy Works, Inc. <dev@democracy.works>"
+
+ARG VERSION=1.1.0
+
+## TODO: How can we architect this such that passing -Dthe.prop=value works as
```

Comment:
🤔 Was this an existing comment that might have gotten moved around? What properties might we need to pass here?

Reply:
No, this was very much me. The PostgreSQL URL, for one! As it is, I have to copy the entire properties file just to change one thing. Take a look at the `docker.properties` file. `Main.java` has the code that would need to change.